client/src/modules/creditors/groups/creditor-groups.ctrl.js

```diff
@@ -15,16 +15,25 @@ function CreditorGroupController($state, CreditorGroup, Notify, Modal) {
   var vm = this;
 
   var uuid = $state.params.uuid;
-
   // global variables
   vm.bundle = {};
   vm.reload = { reload: true };
   vm.state = $state;
+  vm.setOrder = setOrder;
 
   // page state
   vm.isDefaultState = ($state.current.name === 'creditorGroups');
   vm.isUpdateState = ($state.current.name === 'creditorGroups.update' && uuid);
   vm.isCreateState = ($state.current.name === 'creditorGroups.create');
+ 
+  vm.toggleFilter = toggleFilter;
+  vm.sortOptions = [
+    { attribute : 'name', key : 'TABLE.COLUMNS.SORTING.NAME_ASC', reverse : false },
+    { attribute : 'name', key : 'TABLE.COLUMNS.SORTING.NAME_DSC', reverse : true },
+    { attribute : 'created_at', key : 'TABLE.COLUMNS.SORTING.CREATED_DSC', reverse : true },
+    { attribute : 'created_at', key : 'TABLE.COLUMNS.SORTING.CREATED_ASC', reverse : false },
+    { attribute : 'total_debtors', key : 'TABLE.COLUMNS.SORTING.TOTAL_ASC', reverse : true },
```

Comment:
This key looks wrong - does it work?  It probably should talk about `creditors` instead of `debtors`.